tests/bluetooth/tester/nrf5340_hci_ipc_cpunet.conf

```diff
@@ -1,5 +1,14 @@
 # Apply this overlay at hci_ipc controller build
-CONFIG_BT_CTLR_CONN_ISO_LOW_LATENCY_POLICY=y
+
+# Limit counts to ensure reduced RAM usage, compared to values in
+# nrf5340_cpunet_iso-bt_ll_sw_split.conf
+CONFIG_BT_CTLR_ADV_ISO_SET=1
+CONFIG_BT_CTLR_ADV_ISO_STREAM_COUNT=2
```

Comment:
Did you consider applying this to the hci_ipc sample instead? 

There are no defined use cases for ISO/Audio that uses multiple BIGs, so this may be a better default.

Reply:
Not sure if we can consider this a hotfix as it is not causing an CI issues, or otherwise block anyone from working with Zephyr. This is unlikely to get merged before 4.1

Reply:
Dont we want a PTS run this weekend? or we wait for a run after the v4.0 release 2 week later?

Reply:
Please give a thumbs up if you want me to revert the overrides here and instead save RAM by reducing heap size?